pyproject.toml

```
@@ -0,0 +1,38 @@
+[tool.poetry]
+name = "loqusdb"
+version = "2.7.7"
+description = "A simple observation count database"
+authors = ["Your Name <you@example.com>"]
+license = "MIT"
+readme = "README.md"
+include = [
+  "README.md",
+  "loqusdb/**/*",
+  "tests/**/*"
+]
+
+[tool.poetry.dependencies]
+python = ">=3.9,<3.11"
+cyvcf2 = "0.30.12"
```

```suggestion
cyvcf2 = "*"
```
Better leave that to a lock file if we dont have a particular reason?

---

Yes, right, I saw an error on the tests when releasing this. Fixing illegal whitespace (space to tab) in the `profile_snv.vcf` and pushed to this branch for good measure. It fixed the test on my end, but maybe there were other issues I'm not aware of?

---

Great! Then I can unfreeze it in the next PR.

---

Updated this PR